src/server/verification/processor/provider/ZoomProvider.js

```diff
@@ -111,17 +120,24 @@ class ZoomProvider implements IEnrollmentProvider {
         // so we'll reject with isNotMatch: true instead
         // to show the error screen on app side immediately
         // notifying about liveness check failed
-        throwCustomException(message, { isNotMatch }, response)
+        throwException(message, { isNotMatch }, response)
       }
 
+      resultBlob = get(response, 'scanResultBlob')
```

Comment:
this seems like a dup of line 108

Reply:
at line 108 we're processing success response, here - an error response.
blob could be present in both cases